backend/src/api/FeedController.ts

```diff
@@ -48,6 +49,10 @@ export default class FeedController {
             perpage: Joi.number().min(1).max(50).default(10),
             format: joiFormat
         });
+        const feedSortingSchema = Joi.object<FeedSortingSaveRequest>({
+            site: Joi.string().regex(/^[a-z\d-]{3,10}$/i).required(),
```

Comment:
We can change it in the future, it would be better to move regexp to constant (see bottom of ApiMiddleware.ts).